src/components/Cart.jsx

```diff
@@ -0,0 +1,38 @@
+
+import bonus from '../data/bonusItems'
+
+export default function Cart({birdFlu, setBirdFlu}) {
+
+let total = birdFlu.length ? birdFlu.map((prev) => prev.amount).reduce((prev, curr) => prev + curr) : 0;
+
+function addShiny(total, bonus){
```

Comment:
How would you implement this logic but use a slice instead of pushing to a new array?